data/jobs.txt

```diff
@@ -1483,6 +1483,169 @@ mission "Core Pirate Attack"
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
 
+mission "Renegade Drones 1"
+	name "Neutralize broken Drone"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drone."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 20
+		"combat rating" > 200
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`Walking past a group of Navy soldiers, you hear one shout out: "Not again, another drone seems to have a malfunction and attacks ships in orbit. Who's turn is it to shoot it down?"`
+			`Nobody answers.`
+			`What do you want to do?`
+			choice
+				`	(Raise your hand and offer to shoot it down.)`
+					goto yes
+				`	(Ignore them and walk on.)`
+					decline
+			label yes
+			`You raise your hand and one of the soldiers notices you.`
+			`"You? Why not. Try not to hurt yourself."`
+			`You got the job.`
+				accept
+	npc evade
+		government "Renegade"
+		personality heroic target vindictive nemesis marked
```

Comment:
Note: Resolved by using Bounty with Support government instead of Renegade. I think not everybody needs to fight the AI, hence for the moment I prefer the Bounty solution.